### PR TITLE
Fix/links fixing

### PR DESCRIPTION
**Pull request by laisspportugal**
**User-Facing Changes**
Updating old links

**Description**
By applying the suggestion giving by @Zed314 we are updating old links.

**Checklist**

- [x] The web version was tested and it is running ok
- [x] The desktop version was tested and it is running ok
- [x] I've updated/created the storybook file(s)
- [x] The release version was updated on package.json files

### CODE REVIEW

**Comment by Simsalabim1234** [comment]
Are you aware that the welcome screen still shows "Foxbox" ? :)

**Comment by laisspportugal** [comment]
Hi @Simsalabim1234 
Yes we are. We are currently working on the logos
Thanks for the hint!

File: packages/studio-base/src/components/AppSettingsDialog/AppSettingsDialog.tsx [hunk 1]
```diff
@@ -144,7 +144,10 @@ const aboutItems = new Map<
     {
       subheader: "Legal",
       links: [
-        { title: "License terms", url: "https://foxglove.dev/legal/studio-license" },
+        {
+          title: "License terms",
+          url: "https://github.com/bmw-software-engineering/lichtblick/blob/main/LICENSE",
+        },
         { title: "Privacy policy", url: "https://foxglove.dev/legal/privacy" },
```
Comment (aneuwald-ctw) [inline packages/studio-base/src/components/AppSettingsDialog/AppSettingsDialog.tsx:151]: The privacy will continue pointing to foxglove?

**Review by aneuwald-ctw — APPROVED** [review]

File: packages/studio-base/src/components/AppSettingsDialog/AppSettingsDialog.tsx (hunk 1 above)
Comment (laisspportugal) [reply packages/studio-base/src/components/AppSettingsDialog/AppSettingsDialog.tsx]: Done, thanks for the hint

**Review by aneuwald-ctw — APPROVED** [review]

**Pull request merged**